Dockerfile-ifort-minimal

```diff
@@ -1,28 +1,4 @@
-# Build stage
-FROM ubuntu:latest
+FROM intel/oneapi-hpckit:2023.0.0-devel-ubuntu22.04
```

Comment:
You are right, the size has significantly changed. I didn't expect it to change that much. Apparently, I compared the wrong numbers. The changes in the **compressed** sizes are the following:

before this MR: `ifort` - `2.46GB`, `nvfortran` - `4.73GB`;
before https://github.com/earth-system-radiation/containers/pull/9/commits/bc74be18a58d13f8c500af9cf994079eb8804683 (the last commit of this MR): `ifort` - `2.07GB` (we can omit installation of the Intel C/C++ compilers and it will give us: `1.25GB`), `nvfortran` - `4.64GB`;
after this MR: `ifort` - `5.24GB`, `nvfortran` - `4.64GB`.

So, the `ifort` image is now much larger but it's not that large if we compare it to the size of `nvfortran` and it saves us quite some maintenance overhead (see https://github.com/earth-system-radiation/containers/pull/9/commits/bc74be18a58d13f8c500af9cf994079eb8804683 and https://github.com/earth-system-radiation/rte-rrtmgp/pull/200/commits/adf74948d24732fe59b3cd7ec45054228207ae53).